crates/core/Cargo.toml

```diff
@@ -47,12 +47,20 @@ thiserror.workspace = true
 async-trait.workspace = true
 url.workspace = true
 
+derive_more.workspace = true
+anstyle.workspace = true
+colorchoice.workspace = true
+
 [dev-dependencies]
 maplit.workspace = true
 httptest.workspace = true
 tempfile.workspace = true
 test-case.workspace = true
 backon.workspace = true
 
+[features]
+serde = []
+arbitrary = []
```

Comment:
Are these features actually being used? Haven't seen any references to them so far